js/script.js

```diff
@@ -0,0 +1,253 @@
+            //make query url
+            var bookURL = "https://api.nytimes.com/svc/books/v3/lists/" +
+                //last year from moment.js
+                lastYear + "-" +
+                //user selected month and day
+                inputMonth + "-" +
+                inputDay +
+                "/hardcover-fiction.json?api-key=PRpWMMh6gJsugEdDfDApTNqb7N7LiRkN";
+            
+            //ajax function using our book URL
+            $.ajax({
+                method:"GET",  
+                url: bookURL,
+            }).then(function(response){
+                //this variable holds a random number between 0 and 9
+                var x = Math.floor((Math.random() * 10));
+                //make a variable holding the bool result at the value of x
+                var bookCover = response.results.books[x].book_image;
+                //add an src of the book cover image 
+                cardImage.attr("src", bookCover);
+                //add an alt
+                cardImage.attr("alt", "book cover image");
+                //apend it to displayed card
+                $(".api-image").append(cardImage);
+            })
+        }
+        //TMDB API Function
+        function getMovieImage() {
+            //make query URL
+            var movieURL = "https://api.themoviedb.org/3/search/movie?api_key=041542fa1899303e8c187b3a599c9d43&language=en-US&query=e&include_adult=false&primary_release_year=" + 
+                //user selected year    
+                inputYear;
+        
+            //ajax function using our movie URL
+            $.ajax({
+                method:"GET",  
+                url: movieURL,
+            }).then(function(response){
+                //find the movie poster in the API and then use that to make the full image url
+                var poster = null;
+                //until there is a non-null value for a movie poster, the api will continue to look
+                for (i=0; i<response.results.length && !poster; i++) {
+                    var current = response.results[i];
+                    if (current.poster_path) {
+                        poster=current.poster_path;
+                        var moviePosterFull = "https://image.tmdb.org/t/p/original" + poster;
+                    }
+                }
+                //add an src of the movie poster
+                cardImage.attr("src", moviePosterFull);
+                //add an alt
+                cardImage.attr("alt", "movie poster image");
+                //apend it to displayed card
+                $(".api-image").append(cardImage);
+            })            
+        }
+    
+        //if the user selects "space", run the getNasaImage function
+        if (cardStyle === "space") {
+            getNasaImage();
+        //if the user selects "book", run the getBookImage function
+        } else if (cardStyle === "book") {
+            getBookImage();
+        // } else if (cardStyle === "paper") {
```

Comment:
you should remove commented out code since it makes it harder to read the code that is actually being executed